.github/dependabot.yml

```diff
@@ -9,7 +9,7 @@ updates:
       - dependabot
 
   - package-ecosystem: "pip"
```

Comment:
Looking at the docs, we may want to try switching the ecosystem to `pip-compile`, which was introduced earlier this year: https://github.blog/changelog/2021-01-19-dependabot-pip-compile-5-5-0-support/

Reply:
i suspect this is not going to work with `pip-compile-multi` but certainly worth a try